webapp/components/SocialMedia/SocialMedia.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <ds-space v-if="user.socialMedia && user.socialMedia.length" margin="large">
+    <base-card class="social-media-bc">
+      <ds-space margin="x-small">
+        <ds-text tag="h5" color="soft">
+          {{ $t('profile.socialMedia') }} {{ userName | truncate(15) }}?
+        </ds-text>
+        <template>
+          <ds-space v-for="link in socialMediaLinks()" :key="link.id" margin="x-small">
+            <a :href="link.url" target="_blank">
+              <img :src="link.favicon" alt="Link:" height="22" width="22" />
+              {{ link.username }}
+            </a>
+          </ds-space>
+        </template>
+      </ds-space>
+    </base-card>
+  </ds-space>
+</template>
+
+<script>
+export default {
+  name: 'social-media',
+  props: {
+    userName: {},
+    user: {},
+  },
+  methods: {
```

Comment:
Can this be a computed property?

Reply:
`500 Internal Error` when using computed